CHANGES/596.misc.rst

```diff
@@ -0,0 +1 @@
+Enforce towncrier requirements for docs to fix the RTD build
```

Comment:
Since this isn't a user facing change, we can skip adding a changelog message for this PR

Reply:
Done!